curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/64646bc0e5a60ea131e03f91.md

```diff
@@ -0,0 +1,181 @@
+---
+id: 64646bc0e5a60ea131e03f91
+title: Step 81
+challengeType: 0
+dashedName: step-81
+---
+
+# --description--
+
+Before you start writing code for the animation, let's take a look at the function you'll use to add an remove elements from the DOM, `setTimeout`.
+
+The `setTimeout` function takes two arguments: a callback function and a number representing the time in milliseconds to wait before executing the callback function.
+
+For example, if you wanted to log `Hello, world!` to the console after `3` seconds, you would write:
+
+```js
+setTimeout(() => {
+  console.log("Hello, world!");
+}, 3000);
+```
+
+Use the `setTimeout` function to add a `1` second delay before the text `Code` to the console. Then see what happens after you enter `5` into the number input and click the `Convert` button.
```

Comment:
```suggestion
Use the `setTimeout` function to add a one second delay before the text `Code` to the console. Then see what happens after you enter `5` into the number input and click the `Convert` button.
```

We aren't referring directly to the code (since the actual value is `1000`), and numbers ten and under are typically written out.